tox.ini

```diff
@@ -26,21 +26,18 @@
 
 [tox]
 envlist =
-  py27,py34,py35,py36,
-  py27-not-relaxed-strides,py34-not-relaxed-strides
+  py35,py36,py37,
+  py37-not-relaxed-strides
 
 [testenv]
 deps=
   nose
```

Comment:
Should probably just remove nose as a dependency, it only supports recent python 3 if patched.